client/src/components/Invite/Invite.scss

```
@@ -0,0 +1,21 @@
+@import 'styles/variables';
+@import 'styles/helpers';
+
+.Invite {
+  h2 {
+    margin-bottom: $spacing-small;
+  }
+  &__LinkBlock {
+    display: flex;
+    margin-top: $spacing-small;
+    align-items: center;
+    [type='text'] {
```

Use the className declared in `Invite.tsx`
```suggestion
    &__Link {
```